lock/src/main/java/org/infinispan/lock/impl/entries/ClusteredLockValue.java

```diff
@@ -31,12 +29,22 @@ public ClusteredLockValue(String requestId, Object owner, ClusteredLockState sta
       this.state = state;
    }
 
-   private ClusteredLockValue() {
-      this.requestId = null;
-      this.owner = null;
-      this.state = ClusteredLockState.RELEASED;
+   @ProtoFactory
+   static ClusteredLockValue protoFactory(String requestId, WrappedMessage wrappedOwner, ClusteredLockState state) {
+      return new ClusteredLockValue(requestId, WrappedMessages.unwrap(wrappedOwner), state);
    }
 
+   @ProtoField(1)
+   public String getRequestId() {
+      return requestId;
+   }
+
+   @ProtoField(2)
+   WrappedMessage getWrappedOwner() {
```

Comment:
double check but the owner is always an `Address`/`JGroupsAddress`.

Reply:
If we're doing this, I think we should probably change the `ClusteredLockImpl` to explicitly use an Address instead of an `object.`

Reply:
It already does as you cannot overwrite the `originator`.

https://github.com/infinispan/infinispan/blob/7206596b4a0258d0b8f1a20d78259e7f270a81ec/lock/src/main/java/org/infinispan/lock/impl/lock/ClusteredLockImpl.java#L82

@karesti can tell us if she plans to change or support other types of owners.